lib/Option.cpp

```diff
@@ -152,6 +155,46 @@ static llvm::cl::opt<bool>
               llvm::cl::desc("Enable experimental C++ support"));
 } // namespace
 
+namespace llvm {
+
+void initializeClspvPasses(PassRegistry &r) {
```

Comment:
This is an odd file to contain the initialization. I'd suggest making a Passes.cpp.

Reply:
Agreed.  I was going to suggest that and then forgot.